### ISSUE TITLE
Wrong colors while playing videos in HDR

### ISSUE THREAD

**Issue by pbasista** [yuliskov/SmartYouTubeTV#414]
When playing videos like [this](https://www.youtube.com/watch?v=1La4QzGeaaQ) one in HDR quality, the picture is too bright or "washed-out".

The Google's YouTube application for Android TV currently does not offer an option to play videos in HDR quality at all, despite the fact that it used to work correctly a few days ago. Now it plays all videos only in SDR quality. As a result, I cannot compare the results directly.

I am not sure if the problem is with some video metadata not being recognized correctly or with those metadata not being present at all in the video.

**Comment by yuliskov** [comment]
On latest beta the same?

**Comment by pbasista** [comment]
> On latest beta the same?

Yes, both the stable version and the latest release version from yesterday behave in the same way.

**Comment by XcellMiguel** [comment]
Hi, sorry to bump this, but I have the same problem on my TV, HDR videos are 99% of the time too bright, washed out.
I think I only could see one HDR video with the right colors, but can't find it anymore.

So for now I have to plug my PC to the TV to view proper YT HDR videos :/
